### ISSUE TITLE
ARM (aarch64) Dockerfile

### ISSUE THREAD

**Issue by bielej** [prologic/tube#6]
Running 
```
$ docker pull prologic/tube
$ docker run -p 8000:8000 -v /path/to/data:/data
```
on aarch64 will result:
```
standard_init_linux.go:211: exec user process caused "exec format error"
```
As far is I know this means that the binary is not compiled for aarch64. [Dockerhub](https://hub.docker.com/r/prologic/tube/tags) confirms it.
`golang:alpine` supports ARM, but manually building `prologic/go-builder` and then building the original Dockerfile using that prebuild image as the builder returns:
```
can't load package: package .: no Go files in /src
```

Could you please prepare a Docker image for ARM?

**Comment by prologic** [comment]
I will prepare and publish ARM Docker images. Thank you!

**Comment by prologic** [comment]
@bielej I've put up a PR that will build at least Docker iamges for the following platforms:

- amd64
- armv6
- armv7
- arm64

As it is time consuming and slow to build ARM* images because it uses QEMU emulation to do so I will probably only build ARM images on new Releases. Is that okay?

**Comment by prologic** [comment]
@bielej Do you mind testing the new ARM images in #9 ?

<img width="1312" alt="Screen Shot 2020-04-04 at 11 27 16" src="https://user-images.githubusercontent.com/1290234/78415770-53ed6f80-7667-11ea-9e11-ba48e37d5daf.png">

Thanks! 😀

**Comment by bielej** [comment]
Turns out that this issue may no be related to CPU architecture:
```
[root@rpi]› docker inspect prologic/tube:snapshot | grep Architecture
        "Architecture": "arm64",
[root@rpi]› docker run -p 8001:8000 -v /var/tube:/data prologic/tube:snapshot
standard_init_linux.go:211: exec user process caused "exec format error"
[root@rpi]› uname -a
Linux rpi 5.3.0-1018-raspi2 #20-Ubuntu SMP Mon Feb 3 19:45:46 UTC 2020 aarch64 aarch64 aarch64 GNU/Linux
```

**Comment by prologic** [comment]
Just curious; Are you able to build from source on your RPi? I currently don't have one handy.

**Comment by bielej** [comment]
Yes, after installing `golang` and `golang-rice` I was able to build from source.

**Comment by prologic** [comment]
Okay no problems. Thanks for testing the images; but looks like they still need some work before I actually merge this PR and publish them. Leave it with me I'll get it fixed by today.

**Comment by prologic** [comment]
No we will still do this. I have unfortunately very little bandwidth myself right now due to my day job. If the community can help out here that would be awesome!

**Comment by luandro** [comment]
Great work on supporting ARM :rainbow: Any news on supporting aarch64?

**Comment by prologic** [comment]
None yet. Finding free time is hard :) PRs welcome!
